frontend/language/src/nb.json

```diff
@@ -1735,6 +1735,7 @@
   "ux_editor.options.tab_code_list": "Velg kodeliste",
   "ux_editor.options.tab_manual": "Sett opp egne alternativer",
   "ux_editor.options.tab_referenceId": "Angi referanse-ID",
+  "ux_editor.options.tab_referenceId_alert_title": "Du har allerede referert til en kodeliste. Skriver du inn en ID, vil referansen bli slettet.",
```

Comment:
I know this text is formatted as is due to the similar text key above, but we should be consistent with the pattern and adapt both to: `...reference_id_...` 🙈